code/game/machinery/kitchen/gibber.dm

```diff
@@ -199,21 +200,19 @@
 			slab_type = critter.meat_type
 	else if(istype(src.occupant,/mob/living/carbon/human))
 		var/mob/living/carbon/human/H = occupant
-		slab_name = src.occupant.real_name
-		slab_type = H.species.meat_type
+		slab_type = H.isSynthetic() ? /obj/item/stack/material/steel : H.species.meat_type

```

Comment:
`meat_type` should be set properly instead of hard-coding a type for synthetics; there's no case where this would be needed that couldn't be served by properly setting `meat_type`.